### PR TITLE
bf: CLDSRV-513 fix request logger for batchDelete

### PR DESCRIPTION
**Pull request by jonathan-gramain**
Arsenal's `DataWrapper.batchDelete()` now already creates a request logger on which it calls `end()` to get the elapsed time. So as there's no need to create one before the call, remove the corresponding code.

Note that the main fix is the arsenal version bump which, by creating a request logger, fixes naturally the forgotten case in `checkHashMatchMD5`.

Also bump werelogs dependency.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
     "@hapi/joi": "^17.1.0",
-    "arsenal": "git+https://github.com/scality/arsenal#7.10.58",
+    "arsenal": "git+https://github.com/scality/arsenal#01294a6efdee7a5de2c69d700a0fa855dc30470d",
```
Comment (ghost) [inline package.json]: Just a reminder to bump the version before merging

**Review by ghost — APPROVED** [review]

**Review by nicolas2bert — APPROVED** [review]

**Comment by jonathan-gramain** [comment]
ping

**Comment by jonathan-gramain** [comment]
ping

**Pull request merged**